### ISSUE TITLE
error in impala-load-store_sales.sh - int vs bigint

### ISSUE THREAD

**Issue by bradcupit** [cloudera/impala-tpcds-kit#8]
When I run impala-load-store_sales.sh I get this error:
```
ERROR: AnalysisException: Possible loss of precision for target table 'tpcds_parquet.store_sales'.
Expression 'ss_sold_date_sk' (type: BIGINT) would need to be cast to INT for column 'ss_sold_date_sk'
```
I believe this is because `et_stores_sales.ss_sold_date_sk` is a bigint but `store_sales.ss_sold_date_sk` is just an int.

I changed `store_sales.ss_sold_date_sk` to a bigint in impala-load-store_sales.sh, and the script ran without error.

**Comment by bradcupit** [comment]
Seems related to PR #7

**Comment by dkumaratcloudera** [comment]
Fixed now thanks for catching.

Thanks,
Dileep

**Comment by bradcupit** [comment]
Fixed in commit c26b5a7b80116e3f406c514fbefeee31e5689bc5

Thank you!
